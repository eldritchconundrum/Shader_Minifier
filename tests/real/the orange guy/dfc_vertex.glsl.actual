/* File generated with Shader Minifier 1.1.6
 * http://www.ctrl-alt-test.fr
 */
#ifndef DFC_VERTEX_GLSL_ACTUAL_
# define DFC_VERTEX_GLSL_ACTUAL_

const char *dfc_vertex_glsl =
 "void main()"
 "{"
   "gl_TexCoord[0]=gl_MultiTexCoord0,gl_Position=ftransform();"
 "}";

#endif // DFC_VERTEX_GLSL_ACTUAL_
