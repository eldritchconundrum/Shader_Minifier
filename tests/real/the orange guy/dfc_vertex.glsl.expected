/* File generated with Shader Minifier 1.1.6
 * http://www.ctrl-alt-test.fr
 */
#ifndef DFC_VERTEX_GLSL_EXPECTED_
# define DFC_VERTEX_GLSL_EXPECTED_

const char *dfc_vertex_glsl =
 "void main()"
 "{"
   "gl_TexCoord[0]=gl_MultiTexCoord0,gl_Position=ftransform();"
 "}";

#endif // DFC_VERTEX_GLSL_EXPECTED_
